test/functional/apps/visualize/_input_control_vis.js

```diff
@@ -286,7 +286,7 @@ export default function ({ getService, getPageObjects }) {
         await testSubjects.click('inputControlCancelBtn');
       });
 
-      it('should clear child control dropdown when parent control filter pill removed', async () => {
+      it.skip('should clear child control dropdown when parent control filter pill removed', async () => {
```

Comment:
why skipping this one too?  Noticed it also being flaky?  Or doesn't work without the context of the top one?

Reply:
I was thinking it was dependent on the test above but that is not the case. I will remove the skip for that test